src/models/encrypt.js

```diff
@@ -0,0 +1,24 @@
+const bcrypt = require('bcrypt')
+
+const hashPassword = (plaintextPassword) => {
+  return new Promise((resolve, reject) => {
+    bcrypt.hash(plaintextPassword, 10, (err, hash) => {
+      if (err) reject(Error('error hashing password'))
+      else if (hash) resolve(hash)
```

Comment:
What if there is no hash?

Reply:
bcrypt will do the error checking for us, I will include an else statement with resolve(hash)